recipe/meta.yaml

```diff
@@ -20,12 +20,12 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
 
   run:
-    - python
-    - cdp
+    - python >=3.6
```

Comment:
Should we add <3.8 as a limit, given the output_viewer problem?

Reply:
@chengzhuzhang, we could do that but it depends on if the problem is with `output_viewer` or with `e3sm_diags`.  If it is a problem with `output_viewer`, the python version should be pinned there rather than here.

Reply:
Good point!

Reply:
@xylar would you help with fixing output_viewer recipe?

Reply:
I'll take care of it.

Reply:
@chengzhuzhang, it will be tricky (but would be possible) to get build 0 of `output_viewer` 1.3.2 marked as broken.  I just added python version constraints in build 1.  If we don't mark build 0 as broken, users will still be able to try to install that build with python>=3.8.  If you would like to have it marked as broken, we need to first create an issue on the feedstock: https://github.com/conda-forge/output_viewer-feedstock/issues discussing the problem, then we need to issue a PR to https://github.com/conda-forge/admin-requests requesting that it be marked as broken and pointing to the issue.